relengapi/blueprints/slaveloan/static/slaveloan_details.js

```diff
@@ -13,4 +15,58 @@ function reload_loans() {
         }
     });
 }
-$(reload_loans);
+$(reload_history);
+
+function complete_action(evt){
+    evt.target.disabled = true;
+    var good = function(action) {
+        alertify.success("Marked action as complete");
+        reload_actions();
+        reload_history();
+    };
+    var bad = function(errmsg) {
+        /* re-enable the button so users can try again */
+        evt.target.disabled=false;
+        alertify.error(errmsg);
+    };
+
+    var action_id = evt.target.value;
+    var result = $.ajax({
+        url: _get_manual_actions_base + '/' + action_id,
+        type: 'PUT',
+        contentType: 'application/json; charset=utf-8',
+        data: JSON.stringify({complete: true})
```

Comment:
Hm, maybe you don't want to support un-completing, since the loan process will carry on anyway.